trivia.py

```diff
@@ -359,6 +365,13 @@ def _stop(self, *args):
             self._save_game()
             self.factory.running = False
 
+    def _write_log(self, logfile, f, q):
+        '''
+        Writes broken and reported questions to log file
+        '''
+        with open(config.SAVE_DIR+logfile, 'a') as log:
```

Comment:
This should probably be `os.path.join(config.SAVE_DIR, logfile)` so that it's more portable.